### DISCUSSION TITLE
setup: The faces count are 0, the mesh shape cannot be created   <C++ Source>  modules/bullet/shape_bullet.cpp:431 @ setup()

### DISCUSSION THREAD

**Discussion by RoyalXXX** [godotengine/godot-proposals#9883] [3D]
I get this error when I select CSGPolygon mode "Path" and use_colision "On".
In editor I can see my mesh but not in the game.
I don't understand what this error means.

Godot 3.6 beta 4

**Comment by Calinou** [comment]
This place is for feature suggestions. For bug reports, please open an issue on https://github.com/godotengine/godot. Make sure to upload a minimal reproduction project as well – this is absolutely required for us to be able to help.

**Reply by RoyalXXX** [reply]
I originally wanted to create a thread there, but I saw that there are so many mandatory fields to fill out and you even have to attach a file. I realised that this would take a lot of time and decided to create a topic here.
